include/JSystem/JUtility/JUTNameTab.h

```diff
@@ -3,4 +3,34 @@
 
 #include "dolphin/types.h"
 
+struct ResNTAB {
+    u16 mEntryNum;
+    u16 mPad0;
+    struct Entry {
+        u16 mKeyCode;
+        u16 mOffs;
+    } mEntries[1];
+
+    inline const char* getName(u32 index) const {
+        return ((const char*)mEntries) + ((u16*)mEntries)[index * 2 + 1] - 4;
```

Comment:
```suggestion
        return ((const char*)mEntries) + mEntries[index].mOffs - 4;
```
Or is there a reason to write it the other way?

Reply:
This was probably nonsense I was writing when trying to get a match for the loop. This branch is a couple months old.